CHANGELOG.md

```diff
@@ -36,6 +36,18 @@ call it as a query call. This resolves a potential security risk.
 
 The message "transaction is a duplicate of another transaction in block ...", previously printed to stdout, is now logged to stderr. This means that the output of `dfx ledger transfer` to stdout will contain only "Transfer sent at block height <block height>".
 
+### feat: accept more ways to specify cycle and e8s amounts
+
+`_` can now be to make large numbers more readable. For example: `dfx canister deposit-cycles 1_234_567 mycanister`
+
+Certain suffixes that replace a number of zeros are now supported. The (case insensiteve) suffixes are:
+- `k` for `000`, e.g. `500k`
+- `m` for `000_000`, e.g. `5m`
+- `b` for `000_000_000`, e.g. `50B`
+- `t` for `000_000_000_000`, e.g. `0.3T`
+
+For cycles (but not trillion cycles), an additional `c` or `C` is also acceptable. For example: `dfx canister deposit-cycles 3TC mycanister`
```

Comment:
I don't understand the "(but not trillion cycles") comment. Do you mean it is not allowed for parameters that already expect an amount in trillion cycles?
I agree with this restriction but I'm not sure if this explanation here helps (or rather achieves the opposite). I don't have a strong opinion, though. Maybe only I found it unclear.

Reply:
How about
```suggestion
For cycles (but not trillion cycles such as `dfx ledger fabricate-cycles --t <trillion cycle amount>`), an additional `c` or `C` is also acceptable. For example: `dfx canister deposit-cycles 3TC mycanister`
```

We can also leave out the caveat since I seriously doubt anyone would think about the one place that has a`--t` flag